core/src/main/java/org/apache/accumulo/core/conf/ConfigurationTypeHelper.java

```diff
@@ -180,6 +180,8 @@ public static <T> T getClassInstance(String context, String clazzName, Class<T>
     if (instance == null && defaultInstance != null) {
       log.info("Using default class {}", defaultInstance.getClass().getName());
       instance = defaultInstance;
+    } else if (instance == null) {
+      log.info("Default instance not supplied, returning null");
```

Comment:
This could be collapsed with the previous case, to simplify the logic:

```java
    if (instance == null) {
      log.info("Using default class ({})",
          defaultInstance == null ? null : defaultInstance.getClass().getName());
      instance = defaultInstance;
    }
```

Reply:
Implemented in f5543b2